docs/sphinx/conf.py.in

```diff
@@ -371,5 +371,9 @@ linkcheck_ignore = ['https://imspector.mpibpc.mpg.de',
     'https://support.apple.com/downloads/quicktime',
     'https://support.apple.com/quicktime',
     'https://nifti.nimh.nih.gov/nifti-1/',
-    'https://www.leica-microsystems.com'
+    'https://www.leica-microsystems.com',
```

Comment:
Should we remove this link as per https://github.com/ome/ome-model/pull/66?

Reply:
Good point, I'll push another commit and remove it